<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>621</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>491</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMsg">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>501</y>
      <width>131</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>555</y>
      <width>131</width>
      <height>56</height>
     </rect>
    </property>
    <property name="text">
     <string>FILE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="usermenu">
    <property name="title">
     <string>USER</string>
    </property>
   </widget>
   <widget class="QMenu" name="servermenu">
    <property name="title">
     <string>SERVER</string>
    </property>
   </widget>
   <addaction name="usermenu"/>
   <addaction name="servermenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
